api/src/main/kotlin/edu/wgu/osmt/RoutePaths.kt

```diff
@@ -45,7 +45,7 @@ object RoutePaths {
         const val COLLECTION_ID = "collectionId"
     }
 
-    fun scrubForConfigure(routePath: String): String {
-        return routePath.replace("{uuid}", "*")
+    fun createRegex(routePath: String): String {
```

Comment:
This is a useful clarification, nice! Can we go all the way to "uuidRegex"?

Reply:
Changing to createUuidRegex()